### ISSUE TITLE
K8SSAND-791 ⁃ Cass operator Image 1.7.1 running as root user 

### ISSUE THREAD

**Issue by RoachMJ** [k8ssandra/cass-operator#157]
**What happened?**
A non default user is not utilized in the image. This is a security concern. 

**Did you expect to see something different?**
Expected to see a non-root user such as cass-operator with restricted access. 
**How to reproduce it (as minimally and precisely as possible)**:

**Environment**
kubernetes 1.19

* Cass Operator version:
1.7.1
k8ssandra/cass-operator
sha256:dd57363b07948560e683610aa11715919b78f1c9a2671f0f4a1fb43a8f0f22b9
    `Insert image tag or Git SHA here`
    
    
* Kubernetes version information:
* Kubernetes cluster kind:
* Manifests:

```
insert manifests relevant to the issue
```

* Cass Operator Logs:

```
insert Cass Operator logs relevant to the issue here
```

**Anything else we need to know?**:

┆Issue is synchronized with this [Jira Bug](https://k8ssandra.atlassian.net/browse/K8SSAND-791) by [Unito](https://www.unito.io)
┆friendlyId: K8SSAND-791
┆priority: Medium

**Comment by jsanda** [comment]
Hi @RoachMJ 

This has already been fixed in the `master` branch and will be available in the next release. See https://github.com/k8ssandra/cass-operator/blob/master/Dockerfile.

**Comment by burmanm** [comment]
Fixed in v1.8.0
